packages/compiler/src/core/cli/utils.ts

```diff
@@ -193,3 +193,17 @@ export function handleInternalCompilerError(error: unknown): never {
 function color(text: string, color: Colors) {
   return pc[color](text);
 }
+
+export function parseCliArgsArg(args: string[] = []): Record<string, string> {
```

Comment:
```suggestion
export function parseCliArgsArgOption(args: string[] = []): Record<string, string> {
```
maybe?

and add a doc comment explaining what its trying to parse